modules/php/Models/Player.php

```diff
@@ -518,4 +518,27 @@ public function react($ids, $ctx)
       }
     }
   }
+
+  public function prepareSelection($card, $selectionCards, $playerIds)
+  {
+    $src = $card->getName();
+    $atom = [
+      'state' => ST_SELECT_CARD,
+//      'type' => 'selection_draw' // TODO: Do we actually need 'type'?
+      'msgActive' => clienttranslate('${you} should choose a card to draw because of ${src_name}'),
+//      'msgWaiting' => clienttranslate('${actplayer} has to react to ${src_name}. You may already select your reaction'),
+      'msgInactive' => clienttranslate('${actplayer} should choose a card to draw because of ${src_name}'),
+      'src_name' => $src,
+      'players' => $playerIds,
+      'isPrivate' => false,
+      'src' => $card->jsonSerialize(),
+      'whoPlayed' => $this->id,
+      'selection' => [$selectionCards],
```

Comment:
I don't think you need to store this since you can fetch it live in the argState using the getSelection function above

Reply:
Makes sense, fixed